src/main/telemetry/smartport.c

```diff
@@ -303,13 +560,31 @@ void handleSmartPortTelemetry(void)
         return;
     }
 
+    if(smartPortFrameReceived) {
+        smartPortFrameReceived = false;
+        // do not check the physical ID here again
+        // unless we start receiving other sensors' packets
+        smartPortFrame_t* frame = (smartPortFrame_t*)smartPortRxBuffer;
+        if(frame->frameId == FSSP_MSPC_FRAME) {
+
+            // Pass only the payload: skip sensorId & frameId
+            handleSmartPortMspFrame(smartPortRxBuffer + 2,
```

Comment:
Well, I use emacs with the screen split in 2, which is why ;-) But I can use long lines if that annoys people.